tensorflow/compiler/xla/service/llvm_ir/fused_ir_emitter.cc

```diff
@@ -306,7 +307,8 @@ bool FusedIrEmitter::IsFusedIrEmitterInefficient(
 
   // Check that the code duplication has at most a factor of 15 (where 15 is an
   // arbitrary constant that seems to work).
-  return total > 15 * index_usage_count.size();
+  const int64 index_usage_count_size_times_15 = 15 * index_usage_count.size(); 
+  return total > index_usage_count_size_times_15;
```

Comment:
Let's use a better name instead of `index_usage_count_size_times_15`. How about `max_duplication_size`?

Reply:
updated.